src/commands/Project.ts

```diff
@@ -78,7 +78,7 @@ export class Project extends Command {
    * @params name a string to create the project
    * @return Observable
    */
-  create(name?: string): Observable<any> {
+  create(name?: [string]): Observable<any> {
```

Comment:
Shouldn't name be of type `string[]`? You are declaring a tuple type not an array type here! Please see regarding tuple at https://www.typescriptlang.org/docs/handbook/basic-types.html.